src/cartesian/Brush.tsx

```diff
@@ -497,10 +500,14 @@ export class Brush extends PureComponent<Props, State> {
       height,
     };
 
+    const ariaValueNow = `ID: ${id}, Min value: ${data[startIndex].name}, Max value: ${data[endIndex].name}`;
```

Comment:
can you give an example of what this ends up being? Does this work for every type of chart that can possibly use brush? Any possibility of index access errors?